llvm/runtimes/CMakeLists.txt

```diff
@@ -261,6 +263,7 @@ function(runtime_default_target)
   llvm_ExternalProject_Add(runtimes
                            ${CMAKE_CURRENT_SOURCE_DIR}/../../runtimes
                            DEPENDS ${ARG_DEPENDS}
+                           ${enable_fortran}
```

Comment:
I think `ENABLE_FORTRAN` could be passed below using the `EXTRA_ARGS` mechanism instead. That would be cleaner than using this "global-but-defined-only-later" `${enable_fortran}` variable.

Reply:
done